pxtblocks/fields/field_turnratio.ts

```diff
@@ -36,23 +36,23 @@ namespace pxtblockly {
 
         createLabelDom_(labelText: string) {
             let labelContainer = document.createElement('div');
-            let svg = Blockly.utils.dom.createSvgElement('svg', {
+            let svg = Blockly.utils.dom.createSvgElement<SVGElement>('svg', {
                 'xmlns': 'http://www.w3.org/2000/svg',
                 'xmlns:html': 'http://www.w3.org/1999/xhtml',
                 'xmlns:xlink': 'http://www.w3.org/1999/xlink',
                 'version': '1.1',
                 'height': (FieldTurnRatio.HALF + FieldTurnRatio.HANDLE_RADIUS + 10) + 'px',
                 'width': (FieldTurnRatio.HALF * 2) + 'px'
             }, labelContainer);
-            let defs = Blockly.utils.dom.createSvgElement('defs', {}, svg);
-            let marker = Blockly.utils.dom.createSvgElement('marker', {
+            let defs = Blockly.utils.dom.createSvgElement<SVGElement>('defs', {}, svg);
+            let marker = Blockly.utils.dom.createSvgElement<SVGElement>('marker', {
                 'id': 'head',
                 'orient': "auto",
                 'markerWidth': '2',
                 'markerHeight': '4',
                 'refX': '0.1', 'refY': '1.5'
             }, defs);
-            let markerPath = Blockly.utils.dom.createSvgElement('path', {
+            let markerPath = Blockly.utils.dom.createSvgElement<SVGElement>('path', {
```

Comment:
nit for future (looks like this is more blockly typing side) -- I believe we should be able to use [template literal types](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html) to make it so we don't need to respecify the types each time like this. I suppose maybe overdoing it though depending on how often it is used.